eng/scripts/Automation-Sdk-Generate.ps1

```diff
@@ -56,7 +56,10 @@ Write-Host "Updated autorest.md files for all the changed swaggers. `n"
 
 $packages = @()
 $dotnet = Join-Path $RepoRoot ".dotnet"
```

Comment:
Yes this is the type of logging I was thinking about. Were you able to use it to find the issue?

Reply:
We need to merge this PR first, then run with a PR pointing to this repo in https://github.com/openapi-env-test/azure-rest-api-specs.